Source/WebInspectorUI/UserInterface/Views/ConsoleMessageView.js

```diff
@@ -549,12 +549,11 @@ WI.ConsoleMessageView = class ConsoleMessageView extends WI.Object
 
     _createRemoteObjectIfNeeded(parameter)
     {
-        // FIXME: Only pass RemoteObjects here so we can avoid this work.
         if (parameter instanceof WI.RemoteObject)
             return parameter;
 
         if (typeof parameter === "object")
-            return WI.RemoteObject.fromPayload(parameter, this._message.target);
+            return WI.RemoteObject.fromPayload(this._message.target, parameter);
```

Comment:
This change is not necessary. The argument order of `WI.RemoteObject.fromPayload()` didn't change (and doesn't need to change).